src/snps/__init__.py

```diff
@@ -654,6 +652,19 @@ def determine_sex(
                 return self._determine_sex_X(heterozygous_x_snps_threshold)
             elif chrom == "Y":
                 return self._determine_sex_Y(y_snps_not_null_threshold)
+            elif chrom == "XY":
+                sex_x = snps_df._determine_sex_X(heterozygous_x_snps_threshold)
+                sex_y = snps_df._determine_sex_Y(y_snps_not_null_threshold)
+
```

Comment:
I'm thinking that this might be a clearer way of structuring the logic below:
```
if not sex_y and sex_x:
    # sex_y undetermined, sex_x determined
    return sex_x
elif sex_y and not sex_x:
    # sex_y determined, sex_x undetermined
    return sex_y
elif sex_y == sex_x:
    # sex_y determined, sex_x determined and are equal.
    return sex_y
else:
    # sex_y determined, sex_x determined and are not equal.
    return ""
```